webhooks/github_event_handler.py

```diff
@@ -3,7 +3,7 @@
 """
 import json
 import logging
-import jenkinsapi.jenkins as Jenkins
+import jenkinsapi.jenkins
```

Comment:
Maybe `from jenkinsapi import Jenkins` (+ make the class name uppercase in `jenkinsapi` module)